src/cryptojwt/jwt.py

```diff
@@ -77,27 +81,28 @@ class JWT:
     """The basic JSON Web Token class."""
 
     def __init__(
-        self,
-        key_jar=None,
-        iss="",
-        lifetime=0,
-        sign=True,
-        sign_alg="RS256",
-        encrypt=False,
-        enc_enc="A128GCM",
-        enc_alg="RSA-OAEP-256",
-        msg_cls=None,
-        iss2msg_cls=None,
-        skew=15,
-        allowed_sign_algs=None,
-        allowed_enc_algs=None,
-        allowed_enc_encs=None,
-        allowed_max_lifetime=None,
-        zip="",
+            self,
+            key_jar=None,
+            iss: str="",
+            lifetime: int = 0,
+            sign: bool = True,
+            sign_alg: str = "RS256",
+            encrypt: bool = False,
+            enc_enc: str = "A128GCM",
+            enc_alg: str = "RSA-OAEP-256",
+            msg_cls: MutableMapping = None,
+            iss2msg_cls: Dict[str, str] = None,
+            skew: int = 15,
+            allowed_sign_algs: List[str] = None,
+            allowed_enc_algs: List[str] = None,
+            allowed_enc_encs: List[str] = None,
+            allowed_max_lifetime: int = None,
+            zip: str = "",
+            typ2msg_cls: Dict[str, str] = None
```

Comment:
```suggestion
            iss2msg_cls: Dict[str, str] = {},
            skew: int = 15,
            allowed_sign_algs: List[str] = [],
            allowed_enc_algs: List[str] = [],
            allowed_enc_encs: List[str] = [],
            allowed_max_lifetime: int = 0,
            zip: str = "",
            typ2msg_cls: Dict[str, str] = {}
```

This is my strategy to follow the typing and matching the False/None type at the same time, without any breaking change on any if or true/false test.

It's only a suggestion, not a request for change. Be you at the decision.

Reply:
Used Optional instead